packages/accordions/src/utils/useAccordionContext.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { createContext, useContext, HTMLAttributes, MutableRefObject } from 'react';
+
+export interface IAccordionContext {
```

Comment:
Could this extend `IUseAccordionPropGetters` from `container-accordions` to get some of these values?

Reply:
Yeah that's a great idea, I will bundle the change in a new commit shortly. Thanks!